response_automater/config.py

```diff
@@ -15,6 +15,7 @@ class Settings(BaseSettings):
     # i.e. CHAT-PROMPT-TEMPLATE and OPENAI-API-KEY which are the names of the secrets in Azure key vault.
     # This is different from the other environment variable names which all use snake case.
     chat_prompt_template: str = Field(default="placeholder-value", alias="chat-prompt-template")
+    chat_message_word_limit: int = 300
```

Comment:
I've put this here to avoid embedding an arbitrary number in too many method names/test names/response error messages. 
I don't think it's worth adding to example.env or the real live env vars themselves, as we're not sure what this check will ultimately look like. For example if we were to adopt something like tiktoken it would naturally need to be renamed token_limit rather than word_limit